charts/rancher-csp-adapter/0.1.0+up0.1.0-rc5/values.yaml

```diff
@@ -0,0 +1,19 @@
+debug: false
+
+image:
+  repository: rancher/rancher-csp-adapter
+  tag: v0.1.0-rc5
+  imagePullPolicy: IfNotPresent
+
+global:
+  cattle:
+    systemDefaultRegistry: ""
+  hostNetwork: false

```

Comment:
Why is this a global setting?

Reply:
I think this was copy paste error. I'll let @MbolotSuse chime in. Deleting for now.

Reply:
Log says that this was there when the chart was handed over to me. I'll double check to make sure that we aren't using this, but if it's not being used should be safe to remove.